sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -147,12 +158,12 @@ case class Invoke(
     }
```

Comment:
Is is better to pass `$funcValIsNull` to `${ev.isNull}`? This is because `$funcValIsNull` determines whether `${ev.value}` is `null` or not instead of `$ev.isNull`.